scripts/process-coderabbit-review.mjs

```
@@ -0,0 +1,103 @@
+aW1wb3J0IHsgZXhlY1N5bmMgfSBmcm9tICdub2RlOmNoaWxkX3Byb2Nlc3Mn
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(`@typescript-eslint/no-unused-expressions`)

---

[error] 'aW1wb3J0IHsgZXhlY1N5bmMgfSBmcm9tICdub2RlOmNoaWxkX3Byb2Nlc3Mn' is not defined.

(no-undef)